website/docs/patterns/form-patterns/partials/guidelines/guidelines.md

```diff
@@ -43,65 +39,45 @@ Fields can be further divided into two types: **text fields** and **data fields*
 | [Toggle](/components/form/toggle) | Lorem ipsum | Lorem ipsum |
 -->
 
-### Sets
+### Field group
 
-Fields may be organized in a **set** either horizontally or vertically if they are collecting related information or information that is part of the same object. Within a set there should bea 16px gap either horizontally (creating multiple columns), or vertically between each field.
+If fields are collecting related information or information that is part of the same object, they may be organized in a field group. Within a field group there should be a 16px gap either horizontally, or vertically between each field.
 
 ![Example of fields organized in a set](/assets/patterns/form-patterns/field-set.png =450x*)
 
-!!! Dont
-
-Don't horizontally organize field types in a set that are related but not of the same type, e.g., a text input and a toggle.
-
-![Organizing different field types](/assets/patterns/form-patterns/different-field-type.png)
-!!!
-
-!!! Do
-
-Do stack fields of different types vertically.
-
-![Organizing different field types](/assets/patterns/form-patterns/different-field-type-do.png =500x*)
-!!!
-
-Common sets of fields can include:
+Common sets of field groups can include:
 
-- Credit card information: card number, expiration data, security code
+- Credit card information: card number, expiration date, security code
 - Multi-line address field
 - First and last name
 
 #### Alignment
 
-Fields organized in a horizontal set should be aligned to the baseline of each element to account for fields with helper text.
+Fields organized in a horizontal field group should be aligned to the baseline of each element to account for fields with helper text.
 
 ![Baseline alignment within a set](/assets/patterns/form-patterns/baseline-alignment.png =500x*)
 
-### Button sets
+### Button Sets
 
-Organize buttons based on the [Button Set](/components/button-set) guidelines, e.g., using a 16px horizontal gap between buttons. More specifications and examples can be found in the documentation for [button organization](/patterns/button-organization).
+Organize buttons based on the [Button Set](/components/button-set) guidelines, e.g., using a 16px horizontal gap between buttons. For more specifications and examples, refer to the documentation for [button organization](/patterns/button-organization).
 
 ![Button set](/assets/patterns/form-patterns/button-set.png =500x*)
 
 ## Layout
 
 ### Single-column
 
-We recommend _most_ forms use a single-column layout as it makes the information easier to parse and reinforces the sequential nature of filling out a form.
+We recommend using a single-column layout for forms to streamline data entry. Since a relative width will adjust based on the viewport and container width, we recommend using a maximum and minimum width to ensure a smooth transition at any size.
```

Comment:
> we recommend using a maximum and minimum width to ensure a smooth transition at any size

As a developer, I am not even sure how can I do it (not even with the new layout components)